FocusEntity-Example/FocusEntity-Example.xcodeproj/project.pbxproj

```diff
@@ -327,14 +327,14 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"FocusEntity-Example/Preview Content\"";
-				DEVELOPMENT_TEAM = 278494H572;
+				DEVELOPMENT_TEAM = 9792AFNZU9;
```

Comment:
Could you add a commit with this and PRODUCT_BUNDLE_IDENTIFIER empty? I try to do that with other repos to simplify the example projects

Reply:
Yes, thank you for pointing that out.